### PR TITLE
fix: add pouch version information

### PR DESCRIPTION
**Pull request by zzchun**
Signed-off-by: Zhanchun Zhang <zzchun@zju.edu.cn>

### Ⅰ. Describe what this PR did
Add `Version` and `CommitID` in version.go

### Ⅱ. Does this pull request fix one issue?
NONE

### CODE REVIEW

**Comment by allencloud** [comment]
To be honest, I do not think all the change make sense. The change still shows user incorrect information.

**Comment by zzchun** [comment]
`Version` and `GitCommit` will be replaced while building packages.
This pr is to add `Version` and `GitCommit` field

File: version/version.go [hunk 1]
```diff
@@ -1,16 +1,16 @@
 package version
 
-import "time"
-
 // Version represents the version of pouchd.
-const Version = "0.1.0"
```
Comment (allencloud) [inline version/version.go]: I suggest that we could use `0.3-dev`.

Comment (zzchun) [reply version/version.go]: Good, I'll change to `0.3-dev`

**Comment by allencloud** [comment]
LGTM

**Pull request merged**